CHANGELOG.md

```diff
@@ -17,6 +17,12 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 ## StreamChatUI
 
 ### ✅ Added
+- Apply channel capabilities in UI components [#2747](https://github.com/GetStream/stream-chat-swift/pull/2747)
```

Comment:
Maybe we can make the entry shorter? Interested customers can always open the link for more info.

Reply:
Fine for me 👍 I added more details to avoid opening the PR, but since you both agree, I'll change this 👍